plugins/inputs/http_response/README.md

```diff
@@ -23,6 +23,11 @@ This input plugin will test HTTP/HTTPS connections.
   # {'fake':'data'}
   # '''
 
+  ## Optional : Look for substring in body of the response
+  # response_string_match = "\"service_status\": \"up\""
```

Comment:
rather than the `or`, you should put a list of examples above with two `##` in front of it